### PR TITLE
Fix checking if zero initialize workgroup memory feature is enabled

### PR DESCRIPTION
**Pull request by ziga-lunarg**

### CODE REVIEW

File: layers/shader_validation.cpp [hunk 1]
```diff
@@ -2478,7 +2478,7 @@ bool CoreChecks::ValidateVariables(const SHADER_MODULE_STATE &module_state) cons
         if (storage_class == spv::StorageClassWorkgroup) {
             // If Workgroup variable is initalized, make sure the feature is enabled
             if (insn.len() > 4 &&
-                !enabled_features.zero_initialize_work_group_memory_features.shaderZeroInitializeWorkgroupMemory) {
+                !enabled_features.core13.shaderZeroInitializeWorkgroupMemory) {
```
Comment (ncesario-lunarg) [inline layers/shader_validation.cpp:2481]: Is there any need now for `zero_initialize_work_group_memory_features` in device_state.h? Looks like `core13.shaderZeroInitializeWorkgroupMemory` will get set whether using 1.3 or the extension (if I understand correctly).

**Review by ncesario-lunarg — APPROVED** [review]
One question, but LGTM!

File: layers/shader_validation.cpp (hunk 1 above)
Comment (ziga-lunarg) [reply layers/shader_validation.cpp:2481]: You are correct.

**Review by Tony-LunarG — APPROVED** [review]
LGTM

**Pull request merged**